src/main/java/com/perficient/movies/exceptions/BusinessException.java

```diff
@@ -0,0 +1,7 @@
+package com.perficient.movies.exceptions;
+
+public class BusinessException extends RuntimeException {
+    public BusinessException(String message) {
```

Comment:
be careful went you use this Bussiness Exception, is not a good idea if you want created a message for the users